answers/myFilter.js

```diff
@@ -1,5 +1,11 @@
 const myFilter = (myArray, callback) => {
-  return null;
+  var filteredArray = [];
+  for(i = 0; i < myArray.length; i++){
```

Comment:
You are creating global variable `i` here. Global variables are bad unless you really intended to do one. More info here: https://softwareengineering.stackexchange.com/questions/277279/why-are-globals-bad-in-javascript